terraform/modules/certificates/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "certificate_arn" {
+  value       = aws_acm_certificate.this.arn
+  description = "The arn of the certificate"
+}

```

Comment:
Is there a way we can output what validation records need to be added (rather than needing to go and look them up)?

(N.B. This is a nice to have - only worth a small amount of time)